lib/src/base-styles/_nx-toast.scss

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../base-styles/nx-alert';
+@use '../scss-shared/nx-container-helpers';
+
+.nx-toast__container {
+  @include nx-container-helpers.container-vertical;
+  margin-right: var(--nx-spacing-2x);
+  margin-top: var(--nx-spacing-2x);
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 2;
+}
+
+.nx-toast__container.centered{
+  margin-right: 0;
+  right: unset;
+  left: 50%;
+  transform: translate(-50%);
+}
+
+.nx-toast {
+  @extend .nx-alert;
+  opacity: 0.5;
+  position: relative;
+  top: -16px;
+  width: 500px;
+}
+
+.nx-toast--success {
+  @extend .nx-alert--success;
+}
+
+.nx-toast--warning {
+  @extend .nx-alert--warning;
+}
+
+.nx-toast--error {
+  @extend .nx-alert--error;
+}
+
+.nx-toast--info {
+  @extend .nx-alert--info;
+}
+
+.animate {
+  transition: all 0.3s ease-in-out;
```

Comment:
The animation looks great in the demo!
I wonder for accessibility reasons we should wrap it so it's only active when prefers reduced motion is off:
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Reply:
> The animation looks great in the demo!
I wonder for accessibility reasons we should wrap it so it's only active when prefers reduced motion is off:
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Hmm judging by the example on that mdn page it seems like in might be acceptable to keep the fade and just remove the slide when that media query is active.

Reply:
> > The animation looks great in the demo!
> > I wonder for accessibility reasons we should wrap it so it's only active when prefers reduced motion is off:
> > https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion
> 
> Hmm judging by the example on that mdn page it seems like in might be acceptable to keep the fade and just remove the slide when that media query is active.

I didn't even know about `prefers-reduced-motion`. Thanks @entroform for pointing it out! I have implemented it so that only the fade effect is visible: eb412a8d51d75a327294f94032f86ff7f5139746